packages/sdk/package.json

```diff
@@ -21,7 +21,7 @@
     "test:unit:browser": "karma start --single-run --unit",
     "test:unit:node": "nyc --no-clean -- mocha 'test/unit/{,!(browser)/**/}*.spec.ts' --file src/index.ts -r config/mocha.env.ts --config config/mocharc.node.js",
     "test:unit": "npm run test:unit:node && npm run test:unit:browser",
-    "test:e2e:browser": "node test/e2e/scripts/DownloadSimpleAuth.js && concurrently --kill-others --success first \"node test/e2e/scripts/StartSimpleAuth.js\" \"npm run test:e2e:browser-case\"",
+    "test:e2e:browser": "npm run test:e2e:browser-case",
```

Comment:
minor: `test:e2e:browser-case` can be removed together.

Reply:
Fixed. Thx!